docs/source/conf.py

```diff
@@ -234,41 +233,16 @@
 # If false, no module index is generated.
 #latex_domain_indices = True
 
-# on_rtd is whether we are on readthedocs.org, this line of code grabbed
-# from docs.readthedocs.org
-# NOTE: it is needed to have these lines before load_dbenv()
+# on_rtd is whether we are on readthedocs.org
 on_rtd = os.environ.get('READTHEDOCS', None) == 'True'
 
-if not on_rtd:  # only import and set the theme if we're building docs locally
-    try:
-        import sphinx_rtd_theme
-        html_theme = 'sphinx_rtd_theme'
```

Comment:
You are also removing the setting of the theme I think. Is this ok? And if so, maybe we can also remove the `sphinx-rtd-theme` dependency?

Reply:
Ah, good catch - my intention was not to switch to another theme (which I don't think is a good idea since you want to compare to how it looks on rtd)
I actually should have left `html_theme = 'sphinx_rtd_theme'` in (without condition, since the dependency should always be present).